### ISSUE TITLE
deep learning

### ISSUE THREAD

**Issue by yusufbtanriverdi** [yusuftengriverdi/breast_histopathology_clf#3]
we should create a baseline deep learning models that will extract features. we can use

- [x] [AlexNet](https://arxiv.org/abs/1404.5997)
- [x] How to change loss function in AlexNet (overwriting `model.train()`).
- [x] Resnet50
- [x] DeepLabv3
- [x] FPN
- [x] Small CNN with Global Average Pooling
- [x] Small CNN with FC layers
- [x] Small DNN
- [x] Bi-RNN

**Comment by yusufbtanriverdi** [comment]
thanks hadeel we have enough cnn extractors
